lib/template/processors/hbs.js

```diff
@@ -0,0 +1,45 @@
+var Handlebars = require('handlebars')
+var TerraformError = require("../../error").TerraformError
+
+module.exports = function(fileContents, options){
+
+  /**
+   * Provides support to load partials.
+   *
+   * @usage without locals
+   *  {{partial '../foo.md'}}
+   *
+   * @example with provided locals
+   *  {{partial '../foo.jade' locals='{ "a": "b" }'}}
```

Comment:
Delete `locals=` here for clarity.  I did have an alternate branch with option 3 implemented, if you wanted to provide the "locals" helper as well.  See commit [ef88790](https://github.com/mckramer/terraform/commit/ef8879089095d0a170c49a0fb83123c1b7ff1cad).